packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -23,20 +23,20 @@ const presetOptions = [
     value: 'angular',
     name: 'angular           [a workspace with a single Angular application]'
   },
+  {
+    value: 'angular-nest',
+    name:
+      'angular-nest      [a workspace with a full stack application (Angular + Nest)]'
+  },
   {
     value: 'react',
     name: 'react             [a workspace with a single React application]'
   },
   {
-    value: 'nest-angular',
+    value: 'react-express',
     name:
-      'angular-nest      [a workspace with a full stack application (Angular + Nest)]'
+      'react-express     [a workspace with a full stack application (React + Express)]'
   }
```

Comment:
I understand that react-express and angular-nest are the most common ones, but why not let users decide what combination of frontend+backend they would prefer?